composer/algorithms/gradient_clipping/README.md

```diff
@@ -0,0 +1,128 @@
+# 📎 Gradient Clipping
+
+[\[How to Use\]](#how-to-use) - [\[Suggested Hyperparameters\]](#suggested-hyperparameters) - [\[Technical Details\]](#technical-details) - [\[Attribution\]](#attribution)
+
+ `Computer Vision`, `Natural Language Processing`
+
+Gradient Clipping
```

Comment:
Need some intro text here (or somehwere) on what gradient clipping is, and the supported clipping types. And perhaps something about how to extend to your own clipping type?

Reply:
added some more text here: 7852f42